jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java

```diff
@@ -15,14 +15,44 @@
  */
 package org.jbpm.workbench.pr.client.editors.instance.list.variables.dash;
 
+import static org.dashbuilder.dataset.filter.FilterFactory.equalsTo;
```

Comment:
I believe you got a different code style formatter, please check the latest from here: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration